addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -73,8 +154,160 @@ def _stripe_make_request(self, endpoint, payload=None, method='POST', offline=Fa
             raise ValidationError("Stripe: " + _("Could not establish the connection to the API."))
         return response.json()
 
+    def _get_stripe_request_headers(self, endpoint):
+        """Returns the Stripe API call header
```

Comment:
```suggestion
        """ Return the headers for the Stripe API request.
```
- leading space
- infinitive verb
- header**s**
- trailing dot
- changed "call" by "request" to be consistent with the rest of your docstring